### ISSUE TITLE
Kernel panic: nixos 16.03 (also 16.09 pre) minimal, livecd 64bit while starting systemd-udevd

### ISSUE THREAD

**Issue by apriori** [NixOS/nixpkgs#17735]
## Issue description

While just booting the iso image, my notebook, an hp spectre x360 with an i7-6500U, the kernel panics on startup of systemd-udevd in cpu_idle. The screen turns black and the keyboard leds blink indicating the kernel panic.
### Steps to reproduce

Boot with kernel param "systemd.confirm_spawn=true"
## Technical details
- Nix version: 16.03, 16.09 pre

**Comment by domenkozar** [comment]
@apriori any other information you can share? There must be something special in your case, as it works for most of us.

**Comment by apriori** [comment]
Well, as far as I can tell now, it was caused by acpi. So I launched the image with acpi=off and nomodeset and finally managed to install nixos. Weird enough, after intstalling acpi=off is no longer needed (would have been a deal breaker anyway, still, just for installing it was ok).
So far most is working, although I didn't try bluetooth yet. Modesetting seems to have a race condition with booting X (in gnome 3) sometimes working and sometimes not. If it fails, the screen will turn black on modesetting and a reboot is needed.
